packages/core/src/metadata/directives.ts

```diff
@@ -52,13 +52,23 @@ export interface DirectiveDecorator {
    * ### Declaring directives
    *
    * Directives are [declarables](guide/glossary#declarable).
-   * They must be declared by an NgModule
-   * in order to be usable in an app.
+   * They must be declared as [standalone](guide/standalone-components) or by an NgModule in order
+   * to be usable in an app.
    *
-   * A directive must belong to exactly one NgModule. Do not re-declare
-   * a directive imported from another module.
-   * List the directive class in the `declarations` field of an NgModule.
+   * A directive must either be standalone or declared inside exactly one NgModule.
+   * Do not re-declare a standalone directive or a directive imported from another module.
    *
+   * You can either declare the directive as standalone and import it in the `imports` field of
+   * another standalone component or NgModule:
+   *
+   * ```ts
+   * imports: [
+   *  AppComponent,
+   *  MyDirective
+   * ],
+   * ```
```

Comment:
Let's may be have a full example here and below, so that developers don't confuse the "imports" section here with the one from the NgModule. I think we can also drop the "AppComponent" from there as it's not relevant in this example.
```suggestion
   * ```ts
   * @Component({
   *   standalone: true,
   *   imports: [MyDirective]
   * })
   * class SomeStandaloneComponent {}
   * ```
```

Reply:
for completeness should we also add the selector and a very simple template which uses the directive (as for example `<div my-directive></div>`)?

Reply:
Good idea, let's add a template and a selector as well 👍

Reply:
added :slightly_smiling_face: 

for the selector I put `standalone-component` although it would need a prefix :sweat: , `my-` didn't sit too well with me as I think it's more indicative for the directive we are referring to :confused: 

what do you think? is 'standalone-component' ok? should it be 'my-standalone-component'? (to keep the same prefix), or something else?